### DISCUSSION TITLE
Unexpected Behavior with Glance and Vimium plugin

### DISCUSSION THREAD

**Discussion by luv2code** [zen-browser/desktop#2519] [General]
I am a [vimium](https://github.com/philc/vimium) user and today I tried to use `F` to open a link a new tab. Instead, it opened in a Glance. This is how I found out about Glance, and I was very confused.

Also frustrating because the Glance doesn't have focus. I can't use vimium keybindings to navigate inside the glance, they apply to the parent page instead. There doesn't seem to be a way to switch to the glance.

Anyone else facing this issue, you have to either turn glance off or switch it to a different key (from ctrl+click to alt-click for example)

**Comment by IgnacioArellano** [comment]
Hi, I have this issue with `O` to open my search result in  a new tab.
With  alt+click or shift+click bindings the search open in glance.
With ctrl+click the search open in a new tab as I wanted, but then... `F` opens links in glance. :cry: As it happens to you.

So I end up leaving the thing in alt+click and now I use Ctrl+TAB like a normal human to open a search in a new tab and `O` when I wanted a quick search that can be in glance.
